content/blog/api-console-for-data-services-cloud-console.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Using API Gateway for Data Services Cloud Console
+date: 2021-07-29T14:28:17.779Z
+featuredBlog: false
+priority: 1
+author: Ron Dharma
+authorimage: /img/dscc-icon.png
+thumbnailimage: ""
+tags:
+  - data-services-cloud-console
```

Comment:
I think the correct tag to use to tie the blog post to the upcoming DSCC API platform:
data-services-cloud-console-api

Thsi is the URL for the DSCC platform page (preview): https://deploy-preview-687--hpe-dev-portal.netlify.app/platform/data-services-cloud-console-api/home/ 

This means that once the DSCC platform page will be published, the URL will be:
https://developer.hpe.com/platform/data-services-cloud-console-api/home/

Ron, I believe you changed the title name for the upcoming DSCC platform page from "Data Services Cloud Console API" to "Data Services Cloud Console".
I am going to check with Jay and Didier how we could align the URL for the upcoming DSCC platform page with the name of the Platform page title "Data Services Cloud Console" so the URL would be https://developer.hpe.com/platform/data-services-cloud-console/home/.